zx: 8.2.0 -> 8.2.2

jlbribeiro opened the pull request:
**Diff:** https://github.com/google/zx/compare/8.2.0...8.2.2
**Changelog:**
- https://github.com/google/zx/releases/tag/8.2.1
- https://github.com/google/zx/releases/tag/8.2.2

**Maintainers**: @jlbribeiro (me)

<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

jlbribeiro committed:
zx: 8.2.0 -> 8.2.1

Diff: https://github.com/google/zx/compare/8.2.0...8.2.1
Changelog: https://github.com/google/zx/releases/tag/8.2.1

Temporarily pinning the commit which modifies package.json,
as that affects the version reported by the CLI
(causing `versionCheckHook` to fail).

0f2be5b053b7649fca84c92cd04310b94e297413~ == refs/tags/8.2.1

See:
- https://github.com/google/zx/issues/938
- https://github.com/google/zx/pull/939

emilylange commented:
https://github.com/google/zx/releases/tag/8.2.2 got released an hour or so ago and appears to have fixed the `package.json` mismatch.

Do you have time to bump the PR or do you want it merged right away and open a follow-up PR for `8.2.2`?

jlbribeiro commented:
@emilylange I have the time to re-bump the package, thank you for asking!
(I'll be adding a commit to the PR, to allow individual versions to be fetched from `nixpkgs`; I'll go back to using tags in the next one)

I'll ping you directly after I do so, if you don't mind.

jlbribeiro committed:
zx: 8.2.1 -> 8.2.2

Diff: https://github.com/google/zx/compare/8.2.1...8.2.2
Changelog: https://github.com/google/zx/releases/tag/8.2.2

jlbribeiro commented:
@emilylange Feel free to review again, and thank you!
